sdk/extensions/Microsoft.Extensions.Azure/Directory.Build.props

```diff
@@ -4,4 +4,8 @@
   </PropertyGroup>
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+
+  <ItemGroup Condition="'$(TargetFramework)' != 'net8.0'">
```

Comment:
does this need a comment similar to [this one](https://github.com/Azure/azure-sdk-for-net/pull/48426/files#diff-9830179411f3dce7a3f5c44d91cbc56c7c3fd016bd604636149f1ac4ac64e585R29-R31)?

Reply:
I was able to remove this (I think?) by embedding the ASP.NET Core app into the tests library.